integration-tests/src/codegen/polkadot.rs

```diff
@@ -2445,6 +2450,7 @@ pub mod api {
                 }
             }
         }
+        #[doc = "\n\t\t\tThe [event](https://docs.substrate.io/v3/runtime/events-and-errors) emitted\n\t\t\tby this pallet.\n\t\t\t"]
```

Comment:
I wonder whether we should look at the lines here and strip any common spacing (eg in this case, every line begins with 3 tabs, so perhaps we should remove 3 tabs from each line?

Also, separately, perhaps it makes sense to remove any leading and trailing newlines?

I'm not too bothered if this doesn't happen though, since the documentation isn't visible in the "common" macro usage, and only really becomes useful if a user generates the code manually.